File: app/src/main/java/com/SReMake/Application.java
package com.SReMake;

import org.babyfish.jimmer.client.EnableImplicitApi;
import org.babyfish.jimmer.sql.EnableDtoGeneration;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableDtoGeneration
@EnableImplicitApi
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: app/src/main/kotlin/com/SReMake/Application.kt
package com.SReMake

import org.babyfish.jimmer.client.EnableImplicitApi
import org.babyfish.jimmer.sql.EnableDtoGeneration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableDtoGeneration
@EnableImplicitApi
@EnableScheduling
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
